Website/Portals/_default/HotcakesViews/_default/Views/Checkout/PaymentError.cshtml

```diff
@@ -41,7 +41,7 @@
             <div class="row-fluid">
                 <div class="col-xs-12 text-center margin-top-lg margin-bottom-xl">
                     <a class="btn btn-default" href="@Url.RouteHccUrl(HccRoute.Checkout, new { action = "cancel" })">@Localization.GetString("CancelOrder")</a>
-                    <input type="submit" id="btnsubmit" class="btn btn-primary" value="@Localization.GetString("PlaceOrder")" />
+                    <input type="submit" id="hcTakeOrder" class="btn btn-primary" value="@Localization.GetString("PlaceOrder")" />
```

Comment:
@VAlmea Is this a new update, or are you restoring it back to the `hcTakeOrder` class? What was the original value?

Reply:
I changed the btnsubmit Id to hcTakeOrder in the PaymentError view in order to reuse all the logic from the checkout view. I reviewed all the js files and the btnsubmit Id for the button is not used.